docs/pages/operations-guide.md

```diff
@@ -453,7 +494,7 @@ Looks like we were successful! Now we can run commands inside this container to
 
 ## Querying the Scheduler
 
-The Scheduler exposes several HTTP endpoints which provide information on any current deployment as well as the Scheduler's view of its tasks. For a full listing of HTTP endpoints, see the . The Scheduler endpoints most useful to field diagnosis come from three sections:
+The Scheduler exposes several HTTP endpoints that provide information on any current deployment as well as the Scheduler's view of its tasks. For a full listing of HTTP endpoints, see the . <!-- the... --> The Scheduler endpoints most useful to field diagnosis come from three sections:
```

Comment:
Had meant to put a link to the [API Reference](http://mesosphere.github.io/dcos-commons/swagger-api/)